### PR TITLE
docs: translator guide

### LINKED ISSUE #2039: Add documentation for translators
## Description

We should have some documentation for translators [like this](https://www.torproject.org/getinvolved/translation-overview.html.en) that has:

- [ ] Brief overview of the project with relevant links if they want to learn more (e.g. to the threat model, etc.)
- [ ] Links to our existing glossary/terminology page in the documentation so that translators know how to translate technical terms that may not exist in the target language
- [ ] An explanation of how to make a Weblate account and contribute translations

## User Stories

As a translator unfamiliar with SecureDrop, I want a single page I can go to in the documentation to learn how to get started contributing translations.

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Fixes #2039

A guide for translators

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/development/l10n.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Getting Started with Weblate translations
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: Nit: 'translations' -> 'Translations'

File: docs/development/l10n.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+=========================================
+
+SecureDrop is a whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. For more information, you may be interested to learn about :doc:`what makes SecureDrop unique <../what_makes_securedrop_unique>`.
+
+SecureDrop uses the Weblate platform. This page lists detailed instructions for getting started with the Weblate platform as a translator. Before starting this guide, you should know which LANGUAGE you would like to translate.
```
Comment (dachary) [inline docs/development/l10n.rst]: > Why is LANGUAGE in caps?

Copy paste from somewhere, no reason. Fixed.

Comment (dachary) [inline docs/development/l10n.rst]: > add a header that's "Register for Weblate" 

Done, good idea

File: docs/development/l10n.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+.. _`Weblate registration page`: https://weblate.securedrop.club/accounts/register/
+
```
Comment (dachary) [inline docs/development/l10n.rst]: Updated with 

Who uses SecureDrop ?
---------------------

There are two kind of SecureDrop users: :doc:`Sources <../source>` and :doc:`Journalists <../journalist>`. A source is an individual who wants to communicate securily and anonymously with a journalist. Sources are not expected to be have any technical background. Journalists using SecureDrop have all received proper training and know what a PGP public key is, they understand the basic workflow of SecureDrop.

File: docs/development/l10n.rst (hunk 1 above)
Comment (dachary) [inline docs/development/l10n.rst]: done

File: docs/development/l10n.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+.. _`Weblate dashboard`: https://weblate.securedrop.club/
+
+Choose a language
+-----------------
+
+From your `Weblate dashboard`_:
+
+|Weblate dashboard screenshot|
+
+click the ``Watched translations`` menu and select ``Suggested translations``:
```
Comment (dachary) [inline docs/development/l10n.rst]: done

File: docs/development/l10n.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+|Weblate suggested translations screenshot|
+
+and click the ``translate`` button to the right on the line that shows your native language:
```
Comment (dachary) [inline docs/development/l10n.rst]: done

File: docs/development/l10n.rst [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+Translate a phrase
+------------------
+
+Each string to be translated is shown in a ``Source`` box and you can translate it right below in the ``Translation`` text area. When you are done, click ``Save`` and another string will be proposed.
+
+|Weblate translate screenshot|
+
+For most strings a screenshot is available on the right side to show how it looks in context. The screenshot can be displayed in full size by clicking on it:
+
+|Weblate translate context screenshot|
```
Comment (dachary) [inline docs/development/l10n.rst]: :-)

File: docs/development/l10n.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+A :doc:`glossary <../terminology>` is available to explain the SecureDrop specific terms.
+
+Getting help
+------------
+
+Should you need help, you can either:
```
Comment (dachary) [inline docs/development/l10n.rst]: done

**Comment by dachary** [comment]
The commit was modified to address all comments. I'll show this draft to Erin while at SHA2017 this weekend, to collect feedback.

File: docs/development/l10n.rst [hunk 8]
```diff
@@ -0,0 +1,101 @@
+Getting Started with Weblate Translations
+=========================================
+
+SecureDrop is a whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. For more information, you may be interested to learn about :doc:`what makes SecureDrop unique <../what_makes_securedrop_unique>`.
+
+SecureDrop uses the Weblate platform. This page lists detailed instructions for getting started with the Weblate platform as a translator. Before starting this guide, you should know which language you would like to translate.
+
+Who uses SecureDrop ?
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: Nit: Extra space before '?':

'Who uses SecureDrop ?' -> 'Who uses SecureDrop?'

File: docs/development/l10n.rst [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+---------------------
+
+There are two kind of SecureDrop users: :doc:`Sources <../source>` and :doc:`Journalists <../journalist>`. A source is an individual who wants to communicate securily and anonymously with a journalist. Sources are not expected to be have any technical background. Journalists using SecureDrop have all received proper training and know what a PGP public key is, they understand the basic workflow of SecureDrop.
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: `securily` -> `securely`

File: docs/development/l10n.rst [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+.. |Weblate translate button screenshot| image:: ../images/weblate/weblate5.png
+.. |Weblate translate screenshot| image:: ../images/weblate/weblate6.png
+.. |Weblate translate context screenshot| image:: ../images/weblate/weblate7.png
+
+Frequently Asked Questions
+--------------------------                                                  
+
+* What if the language I want to translate is not on the list?
+
+  You can send a request for a new language in `translation category of the SecureDrop forum`_. But please make sure the language you want is definitely not present!
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: `in` -> `in the`

File: docs/development/l10n.rst (hunk 8 above)
Comment (dachary) [inline docs/development/l10n.rst]: done

**Comment by dachary** [comment]
> check out the documentation guidelines. 

Turns out I ignored almost every advice from the guidelines, not good. Fixed now.

**Comment by dachary** [comment]
> I'll show this draft to Erin while at SHA2017 this weekend, to collect feedback.

I did not get a chance to show that to Erin after all so... we're good to merge and update afterwards.

File: docs/development/l10n.rst (hunk 9 above)
Comment (dachary) [inline docs/development/l10n.rst]: done

File: docs/development/l10n.rst (hunk 10 above)
Comment (dachary) [inline docs/development/l10n.rst]: done

**Comment by dachary** [comment]
modified to address your comments & rebased & repushed !

File: docs/development/l10n.rst [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+instructions for getting started with the Weblate platform as a
+translator. Before starting this guide, you should know which language
+you would like to translate.
+
+Who uses SecureDrop?
+--------------------
+
+There are two kind of SecureDrop users: :doc:`Sources <../source>` and
+:doc:`Journalists <../journalist>`. A source is an individual who
+wants to communicate securily and anonymously with a
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: Looks like this `securily` -> `securely` change didn't make in

**Comment by dachary** [comment]
Nice catch. I'm sure I did fix that securily issue (;-) but apparently not. Repushed after fixing !

File: docs/development/l10n.rst (hunk 11 above)
Comment (dachary) [inline docs/development/l10n.rst]: done

**Comment by redshiftzero** [comment]
Waiting for full test run in PR #2088 to pass, then merging

**Comment by redshiftzero** [comment]
All checks passed in #2088, so merging this in.

**Pull request merged**